cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,20 +1,30 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
+import {MenuContentPage, ProductsListPage, ShoppingCartPage, LoginPage, AddressStepPage,
+  ShippingStepPage, PaymentStepPage} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productsListPage = new ProductsListPage();
+const shoppingCartPage = new ShoppingCartPage();
+const loginPage = new LoginPage();
+const addressStepPage = new AddressStepPage();
+const shippingStepPage = new ShippingStepPage();
+const paymentStepPage = new PaymentStepPage();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("button").contains("Sign in").click();
-    cy.get("button[name=processAddress]").click();
-    cy.get("input[name=cgv]").check();
-    cy.get("button[name=processCarrier]").click();
-    cy.get(".bankwire").click();
-    cy.get("button").contains("I confirm my order").click();
 
-    cy.get("#center_column > div > p > strong").should("have.text", "Your order on My Store is complete.");
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productsListPage.clickAddCartButton();
+    productsListPage.clickProceedCheckoutButton();
+    shoppingCartPage.clickProceedCheckoutButton();
+    loginPage.enterEmailAdress();
+    loginPage.enterPassword();
+    loginPage.clickSignInButton();
+    addressStepPage.clickContinueCheckoutButton();
+    shippingStepPage.checkAcceptTerms();
+    shippingStepPage.clickContinueCheckoutButton();
+    paymentStepPage.clickPayByWire();
```

Comment:
rename the method as payByBankWire or selectBankWire

Reply:
Method renamed to suggested method